backend/de.metas.contracts/src/main/java/de/metas/contracts/commission/commissioninstance/businesslogic/sales/CommissionShare.java

```diff
@@ -47,17 +45,23 @@
  */
 
 @Data
-public class SalesCommissionShare
+public class CommissionShare
 {
-	/** can be null if this share was not yet persisted. */
-	private final SalesCommissionShareId id;
+	/**
+	 * can be null if this share was not yet persisted.
+	 */
+	private final CommissionShareId id;
 
 	private final CommissionConfig config;
 
 	private final HierarchyLevel level;
 
 	private final Beneficiary beneficiary;
 
+	private final Payer payer;
+
+	private final Boolean isSOTrx;
```

Comment:
consider using SOTrx enum instead of Boolean

Reply:
good point 👍